assets/js/main.js

```diff
@@ -660,139 +588,41 @@ function updateNews() {
   }
 }
 
-function getLabelColor(faction) {
-  switch (faction) {
-  case 'Corpus':
-    return 'label-info';
-  case 'Grineer':
-    return 'label-danger';
-  case 'Infested':
-    return 'label-success';
-  case 'Corrupted':
-    return 'label-warning';
-  default:
-    return 'label-default';
-  }
-}
+function parseRewards(rewardsData) { // eslint-disable-line no-unused-vars
+  const rewards = rewardsData.items.slice();
+  // const rewardsSet = new Set(rewards);
 
-function getProgressBarColor(faction) {
-  switch (faction) {
-  case 'Corpus':
-    return 'corpus-invasion';
-  case 'Grineer':
-    return 'grineer-invasion';
-  case 'Infested':
-    return 'infested-invasion';
-  case 'Corrupted':
-    return 'corrupted-invasion';
-  default:
-    return 'default-invasion';
+  for (const countedItem of rewardsData.countedItems) {
+    const count = countedItem.count > 1 ? `${countedItem.count} ` : '';
+    rewards.push(count + countedItem.type);
+    // rewardsSet.add(countedItem.type);
```

Comment:
could we get rid of the random commented code?

Reply:
Sure we can, just wanted to leave it for when we decide to implement reward filtering.